fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -68,28 +71,29 @@ class CDUInitPage {
         let requestButtonLabel = "INIT";
         let requestEnable = true;
 
-        if (mcdu.simbrief.sendStatus === "REQUESTING") {
+        if (mcdu.simbriefOfpState === "Requested") {
```

Comment:
Can we put these different request states like `"Requested"` into an `Object.freeze`?

Reply:
Done.